pkg/server/container_attach.go

```diff
@@ -60,7 +60,7 @@ func (c *criService) attachContainer(ctx context.Context, id string, stdin io.Re
 	if err != nil {
 		return errors.Wrap(err, "failed to load task")
 	}
-	handleResizing(resize, func(size remotecommand.TerminalSize) {
+	handleResizing(ctx, resize, func(size remotecommand.TerminalSize) {
```

Comment:
the ctx is todo type `context.Background()`😅
and I think the lifecycle is not limited in one attach request.

Reply:
I thought attach is similar with exec, sorry🤣. working on it

Reply:
Hi @fuweid , I've added a ctx with cancel like what did in `execInternal`. PTAL~